src/binding.cc

```diff
@@ -26,89 +27,72 @@
 #define ICONV_CONST
 #endif  // ICONV_CONST
 
-namespace
-{
-
-using v8::Array;
-using v8::Boolean;
-using v8::FunctionTemplate;
-using v8::Handle;
-using v8::Integer;
-using v8::Local;
-using v8::Null;
-using v8::Object;
-using v8::ObjectTemplate;
-using v8::String;
-using v8::Value;
 
+using namespace Napi;
```

Comment:
You don't actually need this line, because the code uses `Napi::...` explicitly everywhere – which is good.